lib/features/context-pad/ContextPad.js

```diff
@@ -239,6 +239,35 @@ ContextPad.prototype.trigger = function(action, event, autoActivate) {
   event.preventDefault();
 };
 
+/**
+ * Trigger context pad action with entry id.
+ *
+ * @param {String} entryId
+ * @param {string} action
+ * @param {Event} event
+ * @param {boolean} [autoActivate=false]
+ */
+ContextPad.prototype.triggerWithId = function(entryId, action, event, autoActivate) {
```

Comment:
Let#s consolditate this. Can'T we make this a low level function, used by `trigger` (instead of duplicating)?

Reply:
> But I think it should still be a public API

Agreed. Just good to re-use things where possible, to prevent bugs in what should be shared behavior.